softvis3d-frontend/src/classes/status/StatusActionQueue.ts

```diff
@@ -21,7 +22,13 @@ export default class StatusActionQueue<T extends StatusAction> {
         console.error("Could not remove action: " + JSON.stringify(action));
     }
 
-    public copyAndUpdate(action: T): StatusActionQueue<T> {
+    /**
+     * As only the content of the action changes, queue has to be recreated
+     * for the @observable to fire an event.
+     *
+     * @returns new instance of StatusActionQueue<T>.
+     */
+    public update(action: T): StatusActionQueue<T> {
```

Comment:
I hope you don't mind me pulling this peace of code back on the table! I just tried to update this, but realised after more than an hour that the issue lies much deeper than the comment suggests:

 * An Update (eg. increaseMax) is triggered
 * The AppStatusStore fetches the Action and Updates the value. _This does not generate a new Object!_
 * The AppStatusStore then asks the Queue to overwrite the action with the same(!) element. `this._queue[i] = sameElement` would not trigger a change in mobx because actually nothing changes.
 * Therefore the complete array is duplicated and then replaced, just to trigger the change.

--------------------------------

I feel this is actually to much compensation and bad/opaque design just to "make it work" with MobX. 
 * Can we use another style/method to display our data in MobX-Conform way?
 * Should we redesign the way LoadActions are stored/managed?